components/form/FormItem.tsx

```diff
@@ -265,6 +265,8 @@ function FormItem<Values = any>(props: FormItemProps<Values>): React.ReactElemen
   let variables: Record<string, string> = {};
   if (typeof label === 'string') {
     variables.label = label;
+  } else if (name) {
+    variables.label = String(name);
```

Comment:
Could you help to add a test case about this?

Reply:
Ok, I will add the test case as soon as possible ~